waf/types/types.go

```diff
@@ -69,6 +71,7 @@ var (
 	_ error = ErrInvalidRule
 	_ error = ErrInvalidFlow
 	_ error = ErrNoRule
+	_ error = ErrOutOfMemory
```

Comment:
Is that a new one defined in the libsqreen WAF spec?

Reply:
No, custom for me with the Go reimplementation when `malloc()` or cgo allocation function (`CString()`, etc.) return nil.